vocabularies/Common.xml

```diff
@@ -223,6 +223,14 @@
         <Annotation Term="Core.Description" String="The annotated set or collection contains at most this number of items" />
       </Term>
 
+      <Term Name="skip" Type="Edm.Int64" AppliesTo="Collection">
+        <Annotation Term="Core.IsInstanceAnnotation"/>
+        <Annotation Term="Core.Description" String="The server assumed this value for $skip when it determined the annotated collection"/>
```

Comment:
Sounds like the annotation would even be expected if $skip was explicitly given? Or does "assume" intend to rule this out?

P.S.: "The query option sap-skiplocation=<entity URI>[,<n>] instructs the server to return an entity collection with the number of entries specified by $top, with a $skip value chosen by the server such that the entity whose URI is given in sap-skiplocation is visible, preferably with <n> entities before it." --> where will this be documented? I like it!

Reply:
Yes. If the request contains a `$skip`, the server need not assume a value.